### PR TITLE
Pull request for Desktop version Project

### PR DESCRIPTION
**Pull request by Aseadd**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,377 +2,405 @@
...
+  <main class="toolbar">
+    <div class="nav-menu">
+      <h4 class="welcome"><a href="#">Welcome</a></h4>
+      <ul class="menu-item">
+        <li class="menu-lists"><a href="#">Projects </a></li>
+        <li class="menu-lists"><a href="#">About</a></li>
+        <li class="menu-lists"><a href="#"> Contact</a></li>
+      </ul>
+      <a href="#" class="menu-icon">
+        <i class="fa fa-bars"></i></a>
```
Comment (wuletawwonte) [inline index.html:24]: - [ ] Please hide the hamburger menu in the desktop version. You can use display block to the button selector inside the media query block for the desktop version. 

Screen Shot 2022-07-08 at 21 21 61

File: index.html [hunk 2]
```diff
@@ -2,377 +2,405 @@
...
-    <main class="toolbar">
-        <div class="nav-menu">
-            <a href="#">Welcome</a>
-            <div class="link-list">
-                <a href="#">project</a>
-                <a href="#">About</a>
-                <a href="#">Information</a>
+  <main class="toolbar">
+    <div class="nav-menu">
+      <h4 class="welcome"><a href="#">Welcome</a></h4>
```
Comment (wuletawwonte) [inline index.html:17]: - [ ] Please remove the hover effect of the logo text. It is not included in the design. Every anchor tag in the website has this hover effect. Please remove `a:hover` effect that changes the background color to pink. 

Screen Shot 2022-07-08 at 21 21 62

File: index.html [hunk 3]
```diff
@@ -2,377 +2,405 @@
 <html lang="en">
```
Comment (wuletawwonte) [inline index.html:2]: - [ ] The width of the content of the website is wider than the width of the viewport. So it scrolls horizontally. Please make sure the width of every element in the website doesn't exceed the size of the viewport. I think you should use `vw` to set the width of elements. 

Screen Shot 2022-07-08 at 23 05 07

File: index.html [hunk 4]
```diff
@@ -2,377 +2,405 @@
...
+        <h4 class="indicator">
+        </h4>
+      </div>
+      <div class="grid-work-row1">
+        <div class="card-enable1">
+          <div class="img-placeholder1">
+            <div class="project-text">
+              <div class="project-title">
+                <h3 class="project-name">Multi-Post Stories Gain+Glory</h3>
+                <ul class="tags">
```
Comment (wuletawwonte) [inline index.html:60]: - [ ] The website has got many things that need to be changed to look like the Figma design. Please make sure it resembles the Figma design. 

<p align="center">&nbsp;
 
| Figma design |  your project |
| ---------- | ----------- |
 | Template 3-main | Screen Shot 2022-07-08 at 23 05 08 |
</p>

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi @Aseadd  👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

- [ ] Please add a description of the changes you made in this pull request in the description section of the pull request. 
- [ ] Please remove dektop.html because you are supposed to implement both mobile and desktop versions of the website in the same HTML file using media queries. And the default HTML file that all browsers open in a project is `index.html`.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Aseadd) [reply index.html:24]: Hi @wuletawwonte,

- Hamburger icon hidden from desktop version

File: index.html (hunk 2 above)
Comment (Aseadd) [reply index.html:17]: - Hover color removed

File: index.html (hunk 3 above)
Comment (Aseadd) [reply index.html:2]: - Width managed and horizontal scroller now, no horizontal scroll trough large width

File: index.html (hunk 4 above)
Comment (Aseadd) [reply index.html:60]: - Layout, font-size and type is updated and is sililar with figma

**Comment by Aseadd** [comment]
### Corrections made based on the feedback

- Hide Hamburger Icon
- Make the width fit the window
- Make font-size and layout similar to the figma diesign
- Remove hover color for the header

File: styles.css [hunk 5]
```diff
@@ -1178,3 +1175,328 @@ button {
...
+  left: 10px;
+  top: 6290px;
+}
+
+@media only screen and (min-width: 768px) {
+  .toolbar {
+    width: 1280px;
+    height: 3250px;
+    margin-left: 30px;
+  }
```
Comment (victorokoroji) [inline styles.css]: - [ ] The width of the content of your website is less than the width of the viewport.  Please make sure the width of every element in the website doesn't exceed the size of the viewport. 

Your Design 👇 
image


The issue is as a result of the fixed-width `width: 1280px;` you added. I would recommend you use percentage (%) or VW to dynamically adjust your page to match the viewport. 

Example:  
```
.toolbar {
    width: 100%
  }
```

File: index.html [hunk 6]
```diff
@@ -2,377 +2,405 @@
...
+        <p class="description-text"> I can help you build a product , feature or website Look through some of my
+          work and experience! If you like what you
+          see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div class="get-mycv">
+          <a href="#">
+            <h6 class="btn-cv">Get my Resume </h6>
+          </a>
+        </div>
+      </div>
+      <div class="card">
```
Comment (victorokoroji) [inline index.html:262]: - [ ] Kindly ensure that the spacing between the technologies cards is even. You can use `display: flex` to justify them evenly.

| Figma Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Aseadd , 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Aseadd** [comment]
### Corrections made

- Width of the content aligned to size of viewport
- Spacing between technology cards made even

**Comment by Aseadd** [comment]
*Hi @vickymarz, *

*I have made changes on my portfolio based on your comments can you check
it and send me your feedback, please?*

**Review by muneebulrehman — APPROVED** [review]
Hi @Aseadd , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good followup with previous reviewer.

## Optional suggestions

- There is still some margin on the right side of your application. I see you put a lot of effort into trying to fix it, so I marked it as optional. But I would strongly recommend you fix this issue. This issue is being created by `main` with the class of toolbar. Have a look at it and try to fix it.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**